sdk/core/azure-core/test/e2e/CMakeLists.txt

```diff
@@ -1,48 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should we remove the `ut` subdirectory? It seems unnecessary, as we can move all the unit tests directly under the test directory:
https://github.com/vhvb1989/azure-sdk-for-cpp/tree/remove-e2e-tests/sdk/core/azure-core/test/ut

Reply:
It depends.
In the future, is we add performance tests, we would need to decide if we want to have the perf as a subdirectory of tests, or as a sibling of test folder.  Since both at tests cases, I would have it as a subdirectory in test